### PR TITLE
Newcontext

### PR DESCRIPTION
**Pull request by rahulgandhi0399**
wrote a new context and exported the functions

### CODE REVIEW

File: src/App.tsx [hunk 1]
```diff
@@ -1,25 +1,37 @@
...
+
+
+
 
 function App() {
+ 
+  // const selectedTaskValue  = useContext(selectedTaskContext);
+  // const tasksValue =   useContext(tasksContext);
+  // tasksValue.push("dummy task")
+  // console.log(selectedTaskValue,tasksValue);
```
Comment (krinashah0493) [inline src/App.tsx:17]: Remove if not used. here and everywhere.. no comments for codebase thats not needed

File: src/TaskForm.tsx [hunk 2]
```diff
@@ -0,0 +1,39 @@
+import { ContextType, useContext } from "react";
+import { TodoContext } from './context/tasks'
+
+const TaskForm = () => {
+  const [todos] = useContext(TodoContext)   ; 
+  
+  console.log(todos.setTasks)
```
Comment (krinashah0493) [inline src/TaskForm.tsx:7]: No console logs in the final pr to that is submitted

File: src/TaskForm.tsx [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+ 
+  return (
+    <div>
+      {/* <div> selected task : {selectedTask} </div> */}
+      <div>
+        taskslist:
+        {/* <ul>
```
Comment (krinashah0493) [inline src/TaskForm.tsx:14]: again here .. do not keep commented code

File: src/context/tasks.tsx [hunk 4]
```diff
@@ -0,0 +1,74 @@
+import * as React from "react";
+import { createContext, useContext } from 'react';
+
+
+export type ThemeContextType = {
+  
+  setTasks : (tasksContext: any  ) => void;
```
Comment (krinashah0493) [inline src/context/tasks.tsx:7]: these functions need to be specific as per the behaviour. Do not create afunction for the sake of it. Setter function may return void but getter function needs to return the desired object that it is meant to

**Review by rahulgandhi0399 — COMMENTED** [review]
created a new context to export functions

**Pull request closed without merging**